assets/src/containers/AssignmentPlanningV2.js

```diff
@@ -145,19 +145,19 @@ function AssignmentPlanningV2 (props) {
                             label: 'Current',
                             value: currentGrade,
                             color: 'steelblue',
-                            labelPlacement: 'down'
+                            labelPlacement: 'down1'
```

Comment:
can the names be more like `current_score_down` or `current_down`?

Reply:
They are used for all the lines not just current score